### PR TITLE
add no-skip-assert rule (fixes #16)

### LINKED ISSUE #16: `no-skip-assert` rule
Rule proposal: Ensure that no assertions are skipped.
Similar to the no-skip-test proposal, but for assertions. They are especially "dangerous" as they do not appear in the end output.

## Fail

```js
const test = require('ava');

test('some title', (t) => {
	t.skip.is(1, 1);
});
```
## Pass

```js
const test = require('ava');

test('some title', (t) => {
	t.is(1, 1);
});
```

I already wrote a rule for this, so I'll submit it if this seems like a good idea to you (after some of the other PRs I proposed get merged)

### PR DESCRIPTION
**Pull request by jfmengels**
add no-skip-assert rule (fixes #16) :)

### CODE REVIEW

File: docs/rules/no-skip-assert.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Ensure no assertions are skipped
+
+It's easy to make an assertion skipped with `test.skip.xyz()` and then forget about it.
```
Comment (sindresorhus) [inline docs/rules/no-skip-assert.md]: `test.skip.xyz()` => `t.skip.xyz()`

File: rules/no-skip-assert.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+'use strict';
+var createAvaRule = require('../create-ava-rule');
+
+function nameOfRootObject(node) {
```
Comment (sindresorhus) [inline rules/no-skip-assert.js]: Can you put this in `util.js`?

File: docs/rules/no-skip-assert.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
+## Fail
+
+```js
+const test = require('ava');
```
Comment (sindresorhus) [inline docs/rules/no-skip-assert.md]: `import test from 'ava';`

**Comment by jfmengels** [comment]
Updated!

**Comment by sindresorhus** [comment]
:+1: ✨🌸

**Pull request merged**